recipes/spdlog/all/conanfile.py

```diff
@@ -169,3 +152,9 @@ def package_info(self):
             self.cpp_info.components["libspdlog"].system_libs = ["pthread"]
         if self.options.header_only and self.settings.os in ("iOS", "tvOS", "watchOS"):
             self.cpp_info.components["libspdlog"].defines.append("SPDLOG_NO_TLS")
+
+        if Version(conan_version).major < 2:
```

Comment:
I was under the impression these were allowed but did nothing...
not sure it's worth hiding under an if 🤔

Reply:
they are allowed... for the moment. It's another way to track conan v1 stuff without a TODO.

Reply:
@SpaceIm in our CCI meeting we do not want to add this everywhere since it will be fix in the next 2.0.0 beta 5

Let's not use this for `env_info` and `user_info` for now :)

❤️